frame/vesting/src/benchmarking.rs

```diff
@@ -375,6 +375,49 @@ benchmarks! {
 		);
 	}
 
+	force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 0 .. T::MAX_VESTING_SCHEDULES;
+
+		let source: T::AccountId = account("source", 0, SEED);
+		let source_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(source.clone());
+		T::Currency::make_free_balance_be(&source, BalanceOf::<T>::max_value());
+
+		let target: T::AccountId = account("target", 0, SEED);
+		let target_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(target.clone());
+		
+		// Give target existing locks
+		add_locks::<T>(&target, l as u8);
```

Comment:
Please add an `assert!(l <= u8::MAX.into());` here, otherwise the benchmarking logic could get messed up.  
The `add_locks` could also be re-written to accept an `u32`, but i think that is out of scope.

Reply:
Added, thanks :)